cvm/vllm_proxy/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[project]
+name = "vllm_proxy"
+version = "0.1.0"
+description = ""
+readme = "README.md"
+requires-python = ">=3.12"
+
+dependencies = [
+    "prometheus-client (>=0.23.1,<0.24.0)",
+    "httpx (==0.28.1)",
+    "fastapi[standard]>=0.119.1",
+]
+
+[dependency-groups]
+test = [
+    "pytest (>=8.4.2,<9.0.0)",
+    "requests (>=2.32.5,<3.0.0)",
+]
```

Comment:
Not really important, but more like an open discussion: Is it necessary to be that restrictive in version specifiers here? (e.g. why are we using a specific version of httpx?) I would think a wider range would still work. I'm also wondering if we need that from the start given that we have lock files.
@jfrery would you have any opinion on that? Not only for this, but for other packages as well. I think I haven't been that restrictive myself, thinking that lock files are already making sure we are using latest with a reproducible build.